pgdatadiff/pgdatadiff.py

```diff
@@ -19,7 +19,14 @@ def make_session(connection_string):
 
 class DBDiff(object):
 
-    def __init__(self, firstdb, seconddb, chunk_size=10000, count_only=False):
+    def __init__(
+        self,
+        firstdb,
+        seconddb,
+        chunk_size=10000,
+        count_only=False,
+        check_columns=None
```

Comment:
The default case is not handled if I'm not wrong? Do you check all the columns when value is `None`?

Reply:
Yes, this is done here https://github.com/klaxit/pgdatadiff/blob/6802dccec9d5a7652e31b7651309c97b0b9990c9/pgdatadiff/pgdatadiff.py#L93